nx/test/nx/optional_test.exs

```diff
@@ -1,7 +1,7 @@
 defmodule Nx.OptionalTest do
-  use ExUnit.Case, async: true
+  use ExUnit.Case, async: false
```

Comment:
It's due to the capture_io calls

Reply:
`capture_io` can be async: true in recent Elixir versions as long as you are doing matches with `=~` instead of `==`, as it may include the capture of other tests. I would say this is OK to do here, so I would go with `async: true` and `=~`.

Reply:
I know, but can't the output of another process get interlaced with the current one?